seeds/dex/trades/dex_trades_seed.csv

```diff
@@ -259,10 +255,8 @@ optimism,zipswap,1,2022-02-05,0x111251020222b3efbd71693d4d5ef3f3e3ff9598f0231528
 optimism,zipswap,1,2022-02-12,0x25b8b0f3465265e895c92353e967bd934d86ae6a0ce9a5a7bf45d967868a1582,9,0x4200000000000000000000000000000000000006,1.00158875384491,0xda10009cbd5d07dd0cecc66161fc93d7c9000da1,2961.21850646285
 optimism,zipswap,1,2022-03-06,0xbc6ec237edd3883acd524defb5f1d8635df3610eb195828b4679b3dc9e6e6b0e,4,0x4200000000000000000000000000000000000006,1.89814384488954,0x7f5c764cbc14f9669b88837ca1490cca17c31607,4924.180007
 optimism,zipswap,1,2022-03-01,0xb285876c166ca5a4614c28cb838a1067cf12b8a0f5217fe1021427f7d4571548,4,0x7f5c764cbc14f9669b88837ca1490cca17c31607,5587.487912,0x4200000000000000000000000000000000000006,1.9
-polygon,quickswap,3,2023-02-17,0x8d1bffc369ccde33fff07cb5edaf58bdc4a63217e5e234d8fe1f3908779610ae,105,0x0d500b1d8e8ef31e21c99d1db9a6444d3adf1270,200,0x2791bca1f2de4661ed88a30c99a7a9449aa84174,293.856899
-polygon,quickswap,3,2023-02-17,0x5761ecf943d9ccdec872484d42d45c28f72b3a3760c6da97b573c8a7e2e38af4,192,0xc2132d05d31c914a87c6611c10748aeb04b58e8f,147.193696,0x0d500b1d8e8ef31e21c99d1db9a6444d3adf1270,100
-fantom,beethoven_x,1,2023-02-01,0xb353ea3753bd43f79452a99a5f0b4ba79848eb535406006b2a325e2610cc20c7,10,0xc3bf643799237588b7a6b407b3fc028dd4e037d2,10.79068245,0x21be370d5312f44cb42ce377bc9b8a0cef1a4c83,10.9191819
-fantom,beethoven_x,1,2023-02-01,0x406e3b0bfa2b7289eb7a9c37a87f55d1c1f7b5dc1c90cf638ccf6bc06b73f4f0,43,0xc3bf643799237588b7a6b407b3fc028dd4e037d2,154.8200719,0x21be370d5312f44cb42ce377bc9b8a0cef1a4c83,156.6637268
-fantom,beethoven_x,1,2023-02-01,0x9ed054600393f6ddfd5e9e271ceacc419ea1ea5c5069032f3bccf6e0bbd1057c,9,0x8d11ec38a3eb5e956b052f67da8bdc9bef8abf3e,4.419601846,0x04068da6c83afcfa0e13ba15a6696662335d5b75,4.417833
-fantom,beethoven_x,1,2023-02-01,0x3ad542dcee62446ad4cea36a4606c1dfecdc654d83113509ca43c958737b97b9,44,0x21be370d5312f44cb42ce377bc9b8a0cef1a4c83,1999.4,0x04068da6c83afcfa0e13ba15a6696662335d5b75,1042.833542
-fantom,beethoven_x,1,2023-02-01,0xf73a2c4c02eb03d263d39f5b640229bb9b23f8242a5389b37a5c8aedc428b17e,56,0x21be370d5312f44cb42ce377bc9b8a0cef1a4c83,84.55916959,0xc5e2b037d30a390e62180970b3aa4e91868764cd,550.564336
```

Comment:
It seems dex_trades_seed file damaged mistakenly during the update, I will recover the damaged seed in the next commit.